src/main/java/org/openstreetmap/atlas/checks/commands/AtlasChecksGeoJSONDiffSubCommand.java

```diff
@@ -33,10 +37,19 @@ public AtlasChecksGeoJSONDiffSubCommand()
     @Override
     protected void mapFeatures(final File file, final HashMap map)
     {
-        final JsonObject json = getGson()
-                .fromJson(new JsonReader(new InputStreamReader(file.read())), JsonObject.class);
-        json.get(FEATURES).getAsJsonArray().forEach(feature -> map
-                .put(feature.getAsJsonObject().get(ID).getAsString(), feature.getAsJsonObject()));
+        try (InputStream inputStream = file.isGzipped()
+                ? new GZIPInputStream(new FileInputStream(file.getFile())) : file.read())
+        {
+            final JsonObject json = getGson()
+                    .fromJson(new JsonReader(new InputStreamReader(inputStream)), JsonObject.class);
+            json.get(FEATURES).getAsJsonArray()
+                    .forEach(feature -> map.put(feature.getAsJsonObject().get(ID).getAsString(),
```

Comment:
This is out of the scope of this PR, but I don't like the pattern where we pass in a data structure and populate it like this. 

Somewhere down the line, we should refactor these classes so mapFeatures takes in a file and returns a `HashMap` instead. Also, `HashMap` should have a generic type on it so we don't get gnarly runtime errors.

Reply:
No, this is my doing. It just got wrapped in a try catch, in this PR. I was planning on cleaning this up when I do the refactoring to handle multiple checks.

Reply:
@danielduhh The updates in this PR are only to handle the gzipped files and it would be nice if the refactoring of the method is handled in another PR. I will work with @Bentleysb  to see how we want to do the refactoring as this command will be refactored in rdar://47649519.